src/app/shared/helper-services/update-data.service.ts

```diff
@@ -491,4 +492,25 @@ export class UpdateDataService {
         }
         return heatCascadingTH;
     }
+
+    updateElectricityReductionTreasureHunt(electricityReductionTreasureHunt: ElectricityReductionTreasureHunt): ElectricityReductionTreasureHunt {
+        if (electricityReductionTreasureHunt.baseline) {
+            electricityReductionTreasureHunt.baseline.forEach(reduction => {
+                reduction = this.updateElectricityReduction(reduction);
+            });
+        }
+        if (electricityReductionTreasureHunt.modification) {
+            electricityReductionTreasureHunt.modification.forEach(reduction => {
+                reduction = this.updateElectricityReduction(reduction);
+            });
+        }
+        return electricityReductionTreasureHunt;
+    }
+
+    updateElectricityReduction(electricityReduction: ElectricityReductionData): ElectricityReductionData {
+        if (electricityReduction.userSelectedHP == undefined) {
```

Comment:
Old Th assessments should probably have the userSelectedHP match their existing rated motor power unit. Ex. my TH is in metric and I've entered 100kW